README.md

```diff
@@ -23,14 +23,21 @@ To install panflute, open the command line and type:
 pip install panflute
 ```
 
-Python 3.6+ and PyPy3 are supported (Python 2.7 and Python 3.3-3.5 were supported up to version 1.11.4).
+###### Python version
+
+> You may need to use `pip3`/`python3` explicitly,
+> as `pip`/`python` still means `pip2`/`python2` on some systems.\
+> Python 3.6+ and PyPy3 are currently supported,
+> while Python 2.7 and Python 3.3 - 3.5 were supported up to panflute version 1.11.4.
```

Comment:
why is this section in blockquotes?

also there is a gap in heading level (2 to 6)